packages/ai/core/generate-object/generate-object.test.ts

```diff
@@ -6,6 +6,7 @@ import { setTestTracer } from '../telemetry/get-tracer';
 import { MockLanguageModelV1 } from '../test/mock-language-model-v1';
 import { MockTracer } from '../test/mock-tracer';
 import { generateObject } from './generate-object';
+import { getEffectiveAbortSignal } from '../../../provider-utils/src/get-effective-abort-signal';
```

Comment:
please move into the ai package under util

Reply:
fixed